skt/kernelbuilder.py

```diff
@@ -73,7 +73,15 @@ def clean_kernel_source(self):
 
     def prepare_kernel_config(self):
         """Prepare the kernel config for the compile."""
-        shutil.copyfile(self.basecfg, "%s/.config" % self.source_dir)
+        if self.cfgtype == 'rh-configs':
+            # Build Red Hat configs and copy the correct one into place
+            self.make_redhat_config()
```

Comment:
It would be good to have documentation explaining what config types we have and noting which use the base configs.

If you can't think of somewhere to put them right now quickly, or it's too long to write, could you perhaps post an issue?